harness/determined/common/experimental/experiment.py

```diff
@@ -87,6 +87,8 @@ class Experiment:
             is 100% completion.
         description: (Mutable, string) Description of the experiment.
         notes: (Mutable, str) Notes for the experiment.
+        project_id: (Mutable, int) ID of the project the experiment is a part of.
+        workspace_id: (Mutable, int) ID of the workspace the experiment runs in.
```

Comment:
workspace_id: (Mutable, int) The ID of the workspace associated with the experiment.